packages/@uppy/drop-target/src/index.ts

```diff
@@ -40,8 +62,8 @@ export default class DropTarget extends BasePlugin {
       meta: {
         // path of the file relative to the ancestor directory the user selected.
         // e.g. 'docs/Old Prague/airbnb.pdf'
-        relativePath: file.relativePath || null,
-      },
+        relativePath: (file as any).relativePath || null,
+      } as any,
```

Comment:
Why the double `as any`?

Reply:
The first one is to silence the TS error that `relativePath` may not exist (we know, that's why we have the `|| null`). The second one is to silence the TS error that `{ relativePath: string | null }` is not assignable to `Record<string, unknown>`.